components/package.json

```diff
@@ -6,6 +6,23 @@
             "main.js": "./main.js"
         }
     },
+    "imports": {
+        "globals": {
+            "Components": true
+        }
```

Comment:
Declare that our package or a dependency thereof uses a global variable. Any bundle generated from this package and dependent modules will declare that it needs the specified global.